src/SearchTrips/Trip.tsx

```diff
@@ -0,0 +1,156 @@
+import { Fragment } from "react";
+import { Feature, Leg, TripPattern } from "../types";
+import { ActionPanel, Action, Color, List } from "@raycast/api";
+import { fetchTrip } from "../api";
+import { useCallback, useEffect, useState } from "react";
+import {
+  getTransportIcon,
+  formatAsClock,
+  formatTimeDifferenceAsClock,
+  formatMetersToHuman,
+} from "../utils";
+import Accessory = List.Item.Accessory;
+
+const buildAccessories = (trip: TripPattern, isDetailsVisible: boolean): Accessory[] => {
+  if (isDetailsVisible) {
+    return [
+      {
+        tag: {
+          value: formatTimeDifferenceAsClock(trip.expectedStartTime, trip.expectedEndTime),
+          color: Color.Blue,
+        },
+      },
+    ];
+  }
+  const accessories: Accessory[] = trip.legs
+    .filter((leg) => !(leg.mode === "foot" && leg.fromPlace.name === leg.toPlace.name))
```

Comment:
It's more precise to use stop ID for this. (We should be able to fetch `id` instead of `name` in the query)

```suggestion
    .filter((leg) => !(leg.mode === "foot" && leg.fromPlace.id === leg.toPlace.id))
```

Reply:
`id` unfortunately doesn't exist in `Place`. It seems Place doesn't implement `PlaceInterface` 🤔

Reply:
Ah, my bad. We'll have to use `fromPlace.quay.stopPlace.id` then. (`quay.stopPlace` gives the parent stop place of the quay)

Reply:
I'll pick this one up soon!